api/envoy/config/filter/http/rbac/v2/rbac.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.rbac.v2;
+option go_package = "v2";
+
+import "envoy/config/rbac/v2alpha/rbac.proto";
+
+import "validate/validate.proto";
+import "gogoproto/gogo.proto";
+
+// [#protodoc-title: RBAC]
+// Role-Based Access Control :ref:`configuration overview <config_http_filters_rbac>`.
+
+message RBAC {
+	// Specify the RBAC rules to be applied globally
```

Comment:
While I review this rest of this change, the indent here is funny, and I'm almost certain we used to run clang-format over the proto files which would have fixed this. I also would have thought that https://github.com/envoyproxy/envoy/pull/3450 would have fixed not running clang-format, so I suspect there is something else going on here. Do you mind taking a quick look to see if you can figure out if/whether clang-format is running on these files? cc @htuch @alyssawilk

Reply:
Oh. hmm. I think the issue is because .proto is in the doc suffix list: https://github.com/envoyproxy/envoy/pull/3450/commits/c5d715a22e2a6682b7ac44ff1aa55eaadd3a20f2#diff-2e4b5720c93b5c27bb0c8c7c4312aaa8R17

I think we want to remove it from there and treat it as code?

Reply:
That'd be fine, but it requires some workarounds (e.g. to avoid the C++ namespace checks in .proto) which I didn't have time for.  It shouldn't be a ton of work though.

Reply:
OK no problem I can fix this later today.